libsolidity/interface/CompilerStack.h

```diff
@@ -289,7 +290,8 @@ class CompilerStack: boost::noncopyable
 	std::vector<Source const*> m_sourceOrder;
 	std::map<std::string const, Contract> m_contracts;
 	std::string m_formalTranslation;
-	ErrorList m_errors;
+	ErrorList m_errorList;
```

Comment:
Ah alright, the constructor is not in the header. We might want to move that at some point as that is mostly the convention in Solidity.